app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -7,27 +7,14 @@ tm-page(title='Staking')
       i.search.material-icons search
 
   modal-search(type="delegates" v-if="somethingToSearch")
+
+  //- Tabs
   .delegates-tabs
     .tab(v-for="(tab, i) in tabs",
       :key="'tab-' + i",
       :class="{'tab-selected': i === tabIndex}",
-      @click="tabIndex = 1") {{tab}}
-  .delegates-container
-    tm-data-loading(v-if="delegates.loading && sortedFilteredEnrichedDelegates.length === 0")
-    tm-data-empty(v-else-if="!delegates.loading && delegates.delegates.length === 0")
-    data-empty-search(v-else-if="!delegates.loading && sortedFilteredEnrichedDelegates.length === 0")
-    template(v-else)
-      panel-sort(:sort='sort', :properties="properties")
-      li-delegate(v-for='i in sortedFilteredEnrichedDelegates' :disabled="!userCanDelegate" :key='i.id' :delegate='i')
-
-  .fixed-button-bar(v-if="!delegates.loading")
-    template(v-if="userCanDelegate")
-      .label #[strong {{ shoppingCart.length }}] validators selected
-      tm-btn(id="go-to-bonding-btn" type="link" to="/staking/bond" :disabled="shoppingCart.length === 0" icon="chevron_right" icon-pos="right" value="Next" color="primary")
-    template(v-else)
-      .label(v-if="!delegation.loadedOnce && delegation.loading") Loading delegations...
-      .label(v-else) You do not have any {{bondingDenom}}s to stake.
-      tm-btn(disabled icon="chevron_right" icon-pos="right" value="Next" color="primary")
+      @click="tabIndex = i")
+        router-link(:to="tabs[i]") {{ tab }}
```

Comment:
`:to="My Stake"` will 404 because it doesn't exist in `routes.js` - try `/staking`